pyproject.toml

```diff
@@ -41,6 +41,7 @@ async-lru = "^2.0.4"
 setuptools = "^68.2.0"
 
 [tool.pytest.ini_options]
+norecursedirs = ".* tests/integration/ef_tests/testdata"
```

Comment:
the ethereum foundation tests repo has its own pytests, so i specifically ask it to ignore those files here

Reply:
removed!